atomic

```diff
@@ -683,6 +683,9 @@ def create_parser(help_text):
     verifyp.add_argument("-v", "--verbose", default=False,
                           action="store_true",
                           help=_("Report status of each layer"))
+    verifyp.add_argument("-V", "--validate", default=False,
+                                action="store_true",
+                                help=_("validate the state of an image"))
     return parser
```

Comment:
If we want the default behavior to do this then yes. Since there was other functionality that `verify` does, I thought doing the system image validation would just be an option for `verify`